### DISCUSSION TITLE
Disable Paid Function

### DISCUSSION THREAD

**Discussion by Xtrato01** [YetiForceCompany/YetiForceCRM#15188] [How to] [answered]
Hi , how would i disable a paid module, someone activated by mistake , so now i need to disable it , but i cant seem to find where?

can someone guide m , i do have the latest Verision of YetiForce.

Thank you.

**Comment by mariuszkrzaczkowski** [comment]
which module did you activate?
what's your message in the icon at the top?

**Reply by Xtrato01** [reply]
The module is Outlook plug in..

well this are the messages when i log in ,
![image](https://user-images.githubusercontent.com/90271495/133652407-a9b9b623-e2c6-4edf-ba59-a50b7f2fbbb5.png)
![image](https://user-images.githubusercontent.com/90271495/133652458-f9a85d33-edbc-471e-a7ce-1f0efaa6ef6c.png)

But when i click on it, there is no option to disable
![image](https://user-images.githubusercontent.com/90271495/133652531-bb3996a8-e623-464a-aa77-9ed0edafe0cf.png)

i tried searching trough sql , but obviously not sure where to look. . Please advice

**Reply by Xtrato01** [reply]
o yes, i also disabled it in modules:

![image](https://user-images.githubusercontent.com/90271495/133652847-85ea034a-d3be-4c57-8523-b02597d6363a.png)

but nothing happens when disabled.

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
Remove the 'https://appsforoffice.microsoft.com', 'https://ajax.aspnetcdn.com' values from the file: Config/Security.php $allowedScriptDomains
https://doc.yetiforce.com/code/classes/Config-Security.html#property_allowedScriptDomains

**Comment by Xtrato01** [comment]
ok perfect that works , thak you !
